quickstarts/microsoft.mobilenetwork/mobilenetwork-create-full-5gc-deployment/azuredeploy.json

```diff
@@ -45,7 +45,7 @@
     },
     "serviceName": {
       "type": "string",
-      "defaultValue": "Allow_all_traffic",
+      "defaultValue": "Allow-all-traffic",
       "metadata": {
```

Comment:
Reverted the Default Service Name to older format to have the consistence with the existing code. This will be addressed in back-end.